### PR TITLE
Add quantization schema

### PR DESCRIPTION
**Pull request by qcolombet**
Plumb quantization schema up to the load command line so that we can choose which kind of quantization (symmetric/asymmetric) we want to use.

This is related to #1321.

### CODE REVIEW

File: lib/Quantization/Base/Base.cpp [hunk 1]
```diff
@@ -149,6 +150,18 @@ TensorQuantizationParams chooseQuantizationParams(float min, float max) {
   min = std::min(min, 0.f);
   max = std::max(max, 0.f);
 
+  if (schema == quantization::Schema::Symmetric) {
```
Comment (rdzhabarov) [inline lib/Quantization/Base/Base.cpp]: can you add some tests to make sure that you'll get the 0 offsets

Comment (qcolombet) [reply lib/Quantization/Base/Base.cpp]: Good point! Forgot to do that after testing the loader.

Comment (qcolombet) [reply lib/Quantization/Base/Base.cpp]: @rdzhabarov, I've added a test case for the quantization schema if you want to take a look.

[the remark this answers is not in the document]
Comment (qcolombet) [reply lib/Quantization/Base/Base.cpp]: Done

File: tests/unittests/quantizationTest.cpp [hunk 2]
```diff
@@ -407,6 +407,113 @@ TEST(Quantization, optimizeRescaleQuantize) {
...
+      chooseQuantizationParams(-3.0, 3.0, quantization::Schema::Asymmetric);
+  // Dequantization formula is scale(X - offset).
+  // So in theory, we should get
+  // 1. scale(-128 - offset) == -3.0
+  // 2. scale(127 - offset) == 3.0
+  // Given scale != 0, #1 + #2 gives scale(-128 + 127 - 2*offset) == 0.0
+  // offset == -1 / -2 == 0.5
+  // Then #2 or #1, gives scale == 3.0 / 127.5.
+  // However, when we get symmetric ranges (i.e., [-X; X]),
+  /// we actually force the zero point to map to 0.
```
Comment (rdzhabarov) [inline tests/unittests/quantizationTest.cpp]: nit: ///

**Comment by jfix71** [comment]
Perhaps add documentation about this to the docs? Either here or in another PR.

**Pull request merged**